modules/collabora_online_group/src/Plugin/Group/RelationHandler/CollaboraPermissionProvider.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\collabora_online_group\Plugin\Group\RelationHandler;
+
+use Drupal\groupmedia\Plugin\Group\RelationHandler\GroupMediaPermissionProvider;
+
+/**
+ * Provides Collabora permissions for group.
+ */
+class CollaboraPermissionProvider extends GroupMediaPermissionProvider {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildPermissions(): array {
+    $permissions = $this->parent->buildPermissions();
+
+    /** @see \Drupal\group\Plugin\Group\RelationHandlerDefault\PermissionProvider::buildPermissions() */
+    $provider_chain = $this->groupRelationTypeManager()->getPermissionProvider($this->pluginId);
```

Comment:
I was a bit confused with this at first, but the the `@see` comment leads to a sufficient explanation.
Perhaps even better could be a short comment.

```suggestion
    // Get the top-level decorator.
    /** @see \Drupal\group\Plugin\Group\RelationHandlerDefault\PermissionProvider::buildPermissions() */
    $provider_chain = $this->groupRelationTypeManager()->getPermissionProvider($this->pluginId);
```
Also in core I see some `/* @see ... */` but no `/** @see .. */`.
I see a lot more `// @see ...`, but I know this is not supported by PhpStorm and possibly other tools. So I always write `/* @see ... */` for code references (not so for web links).
```suggestion
    // Get the top-level decorator.
    /* @see \Drupal\group\Plugin\Group\RelationHandlerDefault\PermissionProvider::buildPermissions() */
    $provider_chain = $this->groupRelationTypeManager()->getPermissionProvider($this->pluginId);
```

Reply:
Ok